tools/migrations/2022_5_19_renameWorkflowTargetAddresses.js

```diff
@@ -0,0 +1,24 @@
+import { asyncMap } from 'utils/async';
+
+export const up = async ({ Sequelize, sequelize, models }) => {
+	const { SubmissionWorkflow } = models;
+
+	await sequelize.queryInterface.addColumn('SubmissionWorkflows', 'targetEmailAddresses', {
+		type: Sequelize.JSONB,
+		allowNull: false,
+		defaultValue: [],
+	});
+
+	await asyncMap(
+		await SubmissionWorkflow.findAll(),
+		(submissionWorkflow) => {
+			submissionWorkflow.targetEmailAddresses = [submissionWorkflow.targetEmailAddress];
+			return submissionWorkflow.save();
+		},
+		{ concurrency: 100 },
+	);
+};
+
+export const down = async ({ sequelize }) => {
```

Comment:
Potentially handy tip: if you wanted to, you could also add another function here to remove the old column and then run it by passing the `--fn <name>` flag to the migration script instead of `--up` or `--down`.